CHANGELOG.md

```diff
@@ -50,7 +50,7 @@
     const appID = receipt.appID;
     ```
 + `getProgram` is moved to `@algo-builder/runtime` from `@algo-builder/algob`.
-
++ Modify `balanceOf` in package `@algo-builder/algob` will return amount(number) of asset account holding. If the account does not hold an asset it will return `undefined`.
```

Comment:
```suggestion
+ `balanceOf` in `@algo-builder/algob` package now return amount (number) of asset account holding and won't print them. If the account does not hold an asset it will return 0.
```